internal/pkg/template/templates/environment/cf.yml

```diff
@@ -71,6 +71,15 @@ Resources:
       Configuration:
         ExecuteCommandConfiguration:
           Logging: DEFAULT
+{{- if .Telemetry }}
+      ClusterSettings:
+        - Name: containerInsights
```

Comment:
Will this accidentally disable an enabled container insights during `env upgrade`? What about 
```suggestion
    {{ - if .Telemetry.EnableContainerInsights}}
        - Name: containerInsights
           Value: enabled
     {{- end}}
```
?

Reply:
1. For existing client environments: `.Telemetry` is nil so we have to have a whole `{{- if .Telemetry}}` block in order to not panic while upgrading their environments.
2. For new environments, `.Telemetry` is always not nil and `EnableContainerInsights` is either `true` or `false` so it should be fine.

Does that make sense or am I missing a case?